### PR TITLE
Add tolerance to dlib face identify

### PR DESCRIPTION
**Pull request by gonzalezcalleja**
**Description:**

Documentation for new tolerance parameter.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24497

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/dlib_face_identify.markdown [hunk 1]
```diff
@@ -50,6 +51,12 @@ faces:
   description: List of faces sources.
   required: true
   type: list
+confidence:
+  description: How much distance between faces to consider it a match. Using tolerance values lower than 0.6 will make the comparison more strict.
+  required: false
+  type: integer
```
Comment (frenck) [inline source/_components/dlib_face_identify.markdown]: According to the parent PR, this should be a float.
```suggestion
  type: float
```

File: source/_components/dlib_face_identify.markdown [hunk 2]
```diff
@@ -30,6 +30,7 @@ image_processing:
    faces:
      Jon: /home/hass/jon.jpg
      Bob: /home/hass/bob.jpg
+   confidence: 0.6
```
Comment (frenck) [inline source/_components/dlib_face_identify.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for opening the PR @gonzalezcalleja! 👍 

I've reviewed it and left some comments. Could you please take a look? Awesome! 👍

File: source/_components/dlib_face_identify.markdown (hunk 1 above)
Comment (gonzalezcalleja) [reply source/_components/dlib_face_identify.markdown]: Done! thanks for your comments.

File: source/_components/dlib_face_identify.markdown (hunk 2 above)
Comment (gonzalezcalleja) [reply source/_components/dlib_face_identify.markdown]: Ok, I'll remove confidence from sample configuration.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**